Misc/NEWS.d/next/Core and Builtins/2023-04-12-20-22-03.gh-issue-87729.99A7UO.rst

```diff
@@ -0,0 +1 @@
+Speed up ``super().meth()`` calls.
```

Comment:
Please introduce LOAD_SUPER_ATTR, LOAD_SUPER_ATTR will be exposed to dis user too :)